docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -738,11 +738,24 @@ You can do more comprehensive verification following [Verify Alluxio]({{ '/en/de
 
 The Alluxio UI can be accessed from outside the kubernetes cluster using port forwarding.
 ```console
-$ kubectl port-forward alluxio-master-$i 19999:19999
+$ kubectl port-forward alluxio-master-$i <local-port>:19999
 ```
-Note: `i=0` for the the first master Pod. When running multiple masters, forward port for each
+The command above allocates a port on the local node `<local-port>` and forwards traffic
+on `<local-port>` to port 19999 of pod `alluxio-master-$i`.
+The pod `alluxio-master-$i` does not have to be on the node you are running this command.
+
+> Note: `i=0` for the the first master Pod. When running multiple masters, forward port for each
 master. Only the primary master serves the Web UI.
 
+For example, you are on a node with hostname `master-node-1` and you would like to serve
+the Alluxio master web UI on `master-node-1:8080`.
+Here's the command you can run:
+```console
+[alice@master-node-1 ~] $ kubectl port-forward --address 0.0.0.0 alluxio-master-$i 8080:19999
```

Comment:
@jiacheliu3 i=1 for this example?

Reply:
Actually no, the pod can be running on another node, it doesn't matter.